opg-phpunit-1604/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.service.opg.digital/opg-php-fpm-1604
 
```

Comment:
I would suggest then using the following if this is going to change oftenish

```
ARG PHPUNIT_VERSION="6.4"

RUN cd /tmp && wget https://phar.phpunit.de/phpunit-${PHPUNIT_VERSION}.phar && \
    chmod +x phpunit-${PHPUNIT_VERSION}.phar && mv phpunit-${PHPUNIT_VERSION}.phar /usr/local/bin/phpunit
```

Reply:
Also can we have one command per line please

Reply:
Done. That's a much better idea